src/components/Item/ItemModified.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Can you add flow/doc on the top of this file, like its there for other js files

Reply:
I have added flow/doc comment